content/open-source/apko/getting-started-with-apko.md

```diff
@@ -59,11 +59,11 @@ You should get output similar to this:
  /_/   \_\ |_|     |_|\_\  \___/
 apko
 
-GitVersion:    v0.5.0
-GitCommit:     71cdb2460aece396d00dd4ea47d7d7b5eab3a37d
-GitTreeState:  clean
-BuildDate:     2022-07-18T16:59:54
-GoVersion:     go1.18.3
+GitVersion:    v0.6.0
```

Comment:
This output would be great to turn into a shortcode chainctl blurb. It shows up on a lot of pages

Reply:
The apko version output? hmm I only found it in this page, I think the others are similar but maybe from melange and sigstore apps?